src/Package/Impl/Editors/BaseEditorFactory.cs

```diff
@@ -318,5 +312,30 @@ public class BaseEditorFactory : IVsEditorFactory, IDisposable {
 
         protected virtual void Dispose(bool disposing) { }
         #endregion
+
+        public static void InitKeyBindings(ITextView textView) {
```

Comment:
It only applies to views that host main editor files. I.e. doesn't apply to REPL or History.